pkg/goods/support/host-support-bundle.tmpl.yaml

```diff
@@ -448,3 +606,38 @@ spec:
         - pass:
             when: "false"
             message: "No signs of hostname changes found"
+  - textAnalyze:
+      checkName: Check if localhost resolves to 127.0.0.1
+      fileName: host-collectors/run-host/localhost-ips.txt
+      regex: 'localhost has address 127.0.0.1'
+      outcomes:
+        - fail:
+            when: "false"
+            message: "'localhost' does not resolve to 127.0.0.1 ip address"
+        - pass:
+            when: "true"
+            message: "'localhost' resolves to 127.0.0.1 ip address"
+  - textAnalyze:
+      checkName: Check if SELinux is enabled
+      fileName: host-collectors/run-host/sestatus.txt
+      regex: '(?m)^Current mode:\s+enforcing'
+      ignoreIfNoFiles: true
+      outcomes:
+        - fail:
+            when: "true"
+            message: "SELinux is enabled when it should be disabled for kubernetes to work properly"
+        - pass:
+            when: "false"
+            message: "SELinux is disabled as expected"
+  - textAnalyze:
+      checkName: "Detect Threat Management and Network Security Tools"
+      fileName: host-collectors/run-host/ps-detect-antivirus-and-security-tools.txt
+      regex: '\b(clamav|sophos|esets_daemon|fsav|symantec|mfend|ds_agent|kav|bdagent|s1agent|falcon|illumio|xagt)\b'
+      ignoreIfNoFiles: true
+      outcomes:
+        - fail:
+            when: "true"
+            message: "Antivirus or Network Security tools detected. These tools can interfere with kubernetes operation."
```

Comment:
I think we need to think about the messaging on this one too. Do we want to always tell them to turn these off? If so we should instead say to disable the tool (and reboot?) to resolve this issue.

If however, we're not _sure_ it results in a failure I'm not sure we should have an analyzer for it. Although I think I'm ok with just telling people to turn it off and if they ignore it they hopefully know what they are doing.

Reply:
update with new language